noetic/distribution.yaml

```diff
@@ -4308,11 +4308,6 @@ repositories:
       type: git
       url: https://github.com/eric-wieser/ros_numpy.git
       version: master
-    release:
-      tags:
-        release: release/noetic/{package}/{version}
-      url: https://github.com/eric-wieser/ros_numpy-release.git
-      version: 0.0.4-4
```

Comment:
@sloretz You could just remove the `version` but leave the rest of the `release` entry to avoid having to enter the information during the next `bloom-release` invocation.

Reply:
Thanks, I didn't know that. Deletes just `version` in f8fb17c